org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -318,3 +318,44 @@ $button-border-radius: 2px;
     }
   }
 }
+
+%btn-primary {
```

Comment:
Because this new `%btn-primary` uses the `!important` rule, without it the style don't work.